src/utils/update/beneficial_owners_managing_officers_data_fetch.ts

```diff
@@ -35,19 +37,25 @@ const initialiseBoAndMoUpdateAppData = (appData: ApplicationData) => {
 
 export const retrieveBeneficialOwners = async (req: Request, appData: ApplicationData) => {
   const pscs: CompanyPersonsWithSignificantControl = await getCompanyPsc(req, appData[EntityNumberKey] as string);
+
+  const boPrivateData = await getBoPrivateData(req, appData);
+
   if (pscs) {
```

Comment:
Could also add an exit point here if pscs is undefined or pscs.items is empty

Reply:
And if there are no pscs, would be want to even retrieve the private data? Could switch the order around

Reply:
will switch the order around! thanks!